gulpfile.js

```diff
@@ -25,10 +25,14 @@ var oghliner = require('./index.js');
 
 gulp.task('default', ['build', 'offline']);
 
-gulp.task('build', function(callback) {
+gulp.task('build', ['build-tabzilla'], function(callback) {
   return gulp.src('app/**').pipe(gulp.dest('dist'));
 });
 
+gulp.task('build-tabzilla', function() {
+  return gulp.src('node_modules/mozilla-tabzilla/**').pipe(gulp.dest('app/styles/tabzilla'));
```

Comment:
Copying the whole directory ends up copying (and caching) some files that aren't necessary:

>[13:44:33] Caching static resource "dist/styles/tabzilla/LICENSE" (16.73 kB)
>[13:44:33] Caching static resource "dist/styles/tabzilla/README.md" (4.75 kB)
>[13:44:33] Caching static resource "dist/styles/tabzilla/css/_tabzilla.scss" (846 B)
>[13:44:33] Caching static resource "dist/styles/tabzilla/css/tabzilla.css" (846 B)
>[13:44:33] Caching static resource "dist/styles/tabzilla/index.js" (424 B)
>[13:44:33] Caching static resource "dist/styles/tabzilla/media/img/tabzilla-static-high-res.png" (1.21 kB)
>[13:44:33] Caching static resource "dist/styles/tabzilla/media/img/tabzilla-static.png" (2.23 kB)
>[13:44:33] Caching static resource "dist/styles/tabzilla/package.json" (1.48 kB)

So I would copy just the tabzilla.css file and the image files. That's a bit more brittle, since the required files and their filenames could change in the future. But it reduces the space required to store the files on the server and clients, and it makes the offlining process less brittle, since a failure to offline a single resource can cause the entire process to fail.